src/app/base/components/GlobalSideNav/GlobalSideNav.tsx

```diff
@@ -0,0 +1,410 @@
+                      }
+                      className="l-navigation__link"
+                      id={navLink.label}
+                      to={navLink.url}
+                    >
+                      <span className="l-navigation__link-text">
+                        {navLink.label}
+                      </span>
+                    </Link>
+                  </li>
+                );
+              } else return null;
+            })}
+          </ul>
+        </>
+      );
+    });
+  }
+
+  if (isAuthenticated) {
+    items.push(
+      <ul className="l-navigation__items">
+        <hr />
+        {isAdmin && showLinks ? (
+          <>
+            <li
+              aria-labelledby="Settings"
+              className={`l-navigation__item ${
+                isSelected(path, {
+                  label: "Settings",
+                  url: urls.settings.index,
+                })
+                  ? "is-selected"
+                  : null
+              }`}
+            >
+              <Link
+                aria-current={
+                  isSelected(path, {
+                    label: "Settings",
+                    url: urls.settings.index,
+                  })
+                    ? "page"
+                    : undefined
+                }
+                className="l-navigation__link"
+                id="Settings"
+                to={urls.settings.index}
+              >
+                <Icon light name="settings" />
+                <span className="l-navigation__link-text">Settings</span>
+              </Link>
+            </li>
+            <hr />
+          </>
+        ) : null}
+
+        <li
+          className={`l-navigation__item ${
+            isSelected(path, { label: "", url: urls.preferences.index })
+              ? "is-selected"
+              : null
+          }`}
+        >
+          <ContextualMenu
+            aria-current={
+              isSelected(path, { label: "", url: urls.preferences.index })
+                ? "page"
+                : undefined
+            }
+            aria-labelledby={authUser?.username}
+            className="l-navigation__link is-dark"
+            id={authUser?.username}
+            position="right"
+            toggleAppearance="link"
+            toggleLabel={
+              <>
+                <Icon light name="profile-light" />
+                <span className="l-navigation__link-text">
+                  {authUser?.username}
+                </span>
+              </>
+            }
+          >
+            <ul>
+              <li>
+                <Link to={urls.preferences.index}>Preferences</Link>
+              </li>
+              <li>
+                <Button appearance="link" onClick={() => logout()}>
+                  Log out
+                </Button>
+              </li>
+            </ul>
+          </ContextualMenu>
+        </li>
+        <hr />
+      </ul>
+    );
+  }
+
+  return items;
+};
+
+const GlobalSideNav = (): JSX.Element => {
```

Comment:
Whilst I think I understand the intention, how about we rename this component along with file and directory names to either `Navigation` to match the main `className` more closely for consistency or `AppSideNavigation` to match `AppSidePanel`.

We don't have any other navigation components that it could be confused with AFAIK. 


```suggestion
const Navigation = (): JSX.Element => {
```
```suggestion
const AppSideNavigation = (): JSX.Element => {
```

Reply:
I think let's go with AppSideNavigation, since that won't be directly confused with Vanilla's "Navigation" component.